eyeshade/workers/reports.js

```diff
@@ -74,5 +77,16 @@ exports.initialize = async (debug, runtime) => {
 
   if ((typeof process.env.DYNO === 'undefined') || (process.env.DYNO === 'worker.1')) {
     setTimeout(() => { daily(debug, runtime) }, 5 * 1000)
+    updateBalancesContinuously(runtime)
   }
 }
+
+async function updateBalancesContinuously (runtime) {
```

Comment:
I think continuous refresh is too aggressive

Reply:
perhaps a mechanism to keep it limited at 1 running at a time, that does not run if it is not asked to?

Reply:
I think I'm partial to something more like updating in the following two cases:
1. A settlement report was uploaded
2. Every 6 hours

Reply:
ok. yeah, that can be done